src/time.rs

```diff
@@ -4,28 +4,26 @@ use core::ops::{Add, AddAssign, Sub, SubAssign};
 
 /// Kernel start time, measured in (seconds, nanoseconds) since Unix epoch
 pub static START: RwLock<(u64, u32)> = RwLock::new((0, 0));

```

Comment:
Since `START` is only edited once, might as well make this just a plain static, without a lock. Should be safe to make it a `static mut`.

Reply:
Good point, I'll add that in another commit here.